release/release_logs/2.0.0/stress_tests/stress_test_many_tasks.json

```diff
@@ -3,45 +3,45 @@
         {
             "perf_metric_name": "stage_0_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 6.4732019901275635
+            "perf_metric_value": 6.37376856803894
         },
         {
             "perf_metric_name": "stage_1_avg_iteration_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 20.34087462425232
+            "perf_metric_value": 23.254157304763794
         },
         {
             "perf_metric_name": "stage_2_avg_iteration_time",
             "perf_metric_type": "LATENCY",
-            "perf_metric_value": 53.219086503982545
+            "perf_metric_value": 71.0486969947815
```

Comment:
this seems to be regression? (+20%)

Reply:
https://b534fd88.us1a.app.preset.io/superset/explore/p/Kp57OGg7knO/

Seems there is an obvious culprit. Looking.

Reply:
d7459391b [Dashboard] Increase the RPC timeout for the snapshot API (#28330)
87b7095bc [core] disable memory monitor for specific nightly tests that are fai… (#28786)
deaae4ce6 [metrics] Add breakdowns for when tasks may be blocked during scheduling and in user code (#28624)
524180eb7 [Tune] Raise better error for incompatible `gcsfs` version (#28772)
e030f956e [RLlib] Enable training step for impala and appo and delete last execution plans. (#28481)
c4348c188 [RLlib] @deprecate(error=True|False) escalation. (#28733)
03c7bca13 [RLlib] `Algorithm.add_policy()` should alternatively accept an already instantiated policy object. (#28637)
06068a9a3 [tune/release tests] pin gcsfs version for now. (#28770)
b49007b61 [tune] Right-aline + wrapped abbreviated columns in table output (#28764)
8ae877a5d Revert "Revert "[RLlib] Deprecate legacy callbacks."" (#28766)
9f1ea307f Add Flyte to ecosystem gallery (#28758)
58255c798 Widening range of grpcio versions allowed. (#28623)

Reply:
I would say it's like this https://github.com/ray-project/ray/commit/deaae4ce628e5e0313cf8f0874178919b99fd51f [metrics] Add breakdowns for when tasks may be blocked during scheduling and in user code (https://github.com/ray-project/ray/pull/28624)

Reply:
I was wrong - looks like the regression already happened with https://github.com/ray-project/ray/commit/524180eb7a70f62fd7acd7a8b032c60f3e8a4eba

Reply:
Ok, the weird thing is that the same commit that didn't have regression now has regression when [rerun](https://console.anyscale.com/o/anyscale-internal/projects/prj_FKRmeV5pA6X72aVscFALNC32/clusters/ses_1Fdd9X7yMQHBEa77Uz9J2Cwf)

Previously: 
<img width="1335" alt="image" src="https://user-images.githubusercontent.com/11676094/198190375-d8f4e2a7-1dc3-45e7-9313-206726244caf.png">


But now: 
<img width="711" alt="image" src="https://user-images.githubusercontent.com/11676094/198190464-d9da2888-919e-42af-be27-b2b3e2a899a3.png">

Reply:
So we now have a regression PR for this metric: https://github.com/ray-project/ray/commit/58255c79878b724288a523bf99c97e4257040c3c

However, due to the nature of the PR (grpcio dependency) + the extreme arg sizes (50), we will not make it release blocking, but follow up in https://github.com/ray-project/ray/issues/29765